client/coral-framework/hocs/withMutation.js

```diff
@@ -6,7 +6,25 @@ import flatten from 'lodash/flatten';
 import isEmpty from 'lodash/isEmpty';
 import {getMutationOptions, resolveFragments} from 'coral-framework/services/graphqlRegistry';
 import {store} from 'coral-framework/services/store';
-import {getDefinitionName} from '../utils';
+import {getDefinitionName, getResponseErrors} from '../utils';
+import t from 'coral-framework/services/i18n';
+
+class ResponseErrors extends Error {
+  constructor(errors) {
+    super(`Response Errors ${JSON.stringify(errors)}`);
+    this.errors = errors.map((e) => new ResponseError(e));
+  }
+}
+
+class ResponseError {
+  constructor(error) {
+    Object.assign(this, error);
```

Comment:
I've always wondered. Does our transpiler evaluate `Object.assign` into something that IE can use? `Object.assign` is not supported in any version.

Counterpoint: do we care? total IE usage is down to 3 or 4%